lib/messages.rb

```diff
@@ -0,0 +1,31 @@
+require 'pg'
+
+class Message
+
+    attr_reader :chit
+
+    def initialize(chit:)
+        @chit = chit
+    end
+
+    def self.list
+        if ENV['ENVIRONMENT'] == 'test'
```

Comment:
As setting up the connection is repeated through methods you can create a private method or a class which might be better.